packages/kbn-unified-data-table/src/types.ts

```diff
@@ -45,3 +45,14 @@ export type CustomCellRenderer = Record<
   string,
   (props: DataGridCellValueElementProps) => React.ReactNode
 >;
+
+export type CustomGridColumnsConfiguration = Record<
+  string,
+  ({
+    column,
+    headerRowHeight,
+  }: {
+    column: EuiDataGridColumn;
+    headerRowHeight?: number;
+  }) => EuiDataGridColumn
```

Comment:
**nit**: for a type definition I think there is no need to destructure the properties since they are not used
```ts
export type CustomGridColumnsConfiguration = Record<
  string,
  (props: { column: EuiDataGridColumn; headerRowHeight?: number }) => EuiDataGridColumn
>;
```

Reply:
Good catch, copy paste mistake